src/layers/core/choropleth-layer/choropleth-layer.js

```diff
@@ -210,3 +207,89 @@ export default class ChoroplethLayer extends BaseLayer {
   }
 
 }
+
+// convert geojson to choropleths
```

Comment:
Thanks for adding these. 

The wording 'to choropleths' is a little vague. How about this function converts list of features from a GeoJSON object to a list of GeoJSON polygons (i.e. array of polygons, first is outline, remaining are holes).

Maybe the return type can be specified differently?
  @returns {[Number,Number][][]}

JSDoc requires using "double" star comments to be recognized by tools such as linters and doc generators, i.e
`/** ... */` instead of `//`